### PR TITLE
Add alert commands

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Based on https://github.com/appium/appium-uiautomator2-server/pull/147

### CODE REVIEW

File: lib/commands/alert.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  * @param {AcceptAlertOptions} opts
+  * @throws {InvalidElementStateError} If no matching button, that can accept the alert,
+  *                                    can be found
+  * @throws {NoAlertOpenError} If no alert is present
+  */
+commands.mobileAcceptAlert = async function (opts = {}) {
+  return await this.uiautomator2.jwproxy.command('/alert/accept', 'POST', opts);
+};
+
+commands.postAcceptAlert = async function () {
```
Comment (jlipps) [inline lib/commands/alert.js]: do we need this at all? isn't the api shape the same and we can just proxy?

Comment (mykola-mokhnach) [reply lib/commands/alert.js]: no, because JSONWP endpoints are different

Comment (jlipps) [reply lib/commands/alert.js]: really? this is what i see in basedriver:

```js
  '/wd/hub/session/:sessionId/alert/accept': {
    POST: {command: 'postAcceptAlert'}
  },
  '/wd/hub/session/:sessionId/alert/dismiss': {
    POST: {command: 'postDismissAlert'}
  },
```

Comment (mykola-mokhnach) [reply lib/commands/alert.js]: really

```
  // old alerts
  '/wd/hub/session/:sessionId/alert_text': {
    GET: {command: 'getAlertText'},
    POST: {command: 'setAlertText', payloadParams: {required: ['text']}}
  },
  '/wd/hub/session/:sessionId/accept_alert': {
    POST: {command: 'postAcceptAlert'}
  },
  '/wd/hub/session/:sessionId/dismiss_alert': {
    POST: {command: 'postDismissAlert'}
  },
  // https://w3c.github.io/webdriver/webdriver-spec.html#user-prompts
  '/wd/hub/session/:sessionId/alert/text': {
    GET: {command: 'getAlertText'},
    POST: {command: 'setAlertText', payloadParams: {required: ['text']}}
  },
  '/wd/hub/session/:sessionId/alert/accept': {
    POST: {command: 'postAcceptAlert'}
  },
  '/wd/hub/session/:sessionId/alert/dismiss': {
    POST: {command: 'postDismissAlert'}
  },
```

**Pull request merged**